fpga/Makefile

```diff
@@ -73,6 +73,22 @@ default: $(mcs)
 fpga_dir := $(base_dir)/fpga/fpga-shells/$(FPGA_BRAND)
 fpga_common_script_dir := $(fpga_dir)/common/tcl
 
+#########################################################################################
+# setup misc. sim files
+#########################################################################################
+SIM_FILE_REQS += \
+	$(ROCKETCHIP_RSRCS_DIR)/vsrc/EICG_wrapper.v
+
+# copy files but ignore *.h files in *.f (match vcs)
```

Comment:
Are you trying to match vcs here or are you assuming that all FPGA synthesis tools will support `+incdir` like VCS does and you can do the same thing as you're doing for VCS?  If you're really trying to match VCS, why?

Reply:
This matches what `GenerateSimFiles` did in the past (filter out headers). I guess another justification for doing this is that #871 is using VCS to do FPGA-level simulation.